packages/account/src/Helpers/utils.tsx

```diff
@@ -157,7 +157,7 @@ export const isDocumentTypeValid = (document_type: FormikValues) => {
 export const isAdditionalDocumentValid = (document_type: FormikValues, additional_document_value?: string) => {
     const error_message = documentAdditionalError(additional_document_value, document_type?.additional);
     if (error_message) {
-        return localize(error_message) + getExampleFormat(document_type.additional?.example_format);
+        return localize(error_message) + getExampleFormat(document_type?.additional?.example_format);
```

Comment:
i don't think we need to pass localize here since `documentAdditionalError` function returned localize string.